src/app/components/home-page/home-page.component.ts

```diff
@@ -26,6 +27,10 @@ export class HomePageComponent {
     console.log(`${restaurant.name} added to favorites!`);
   }
   
+  navigateToRestaurant(id: string): void {
+    this.router.navigate([`/restaurants/${id}`]);
```

Comment:
use your constant here:
```suggestion
    this.router.navigate([`/${RoutingConstants.Restaurants}/${id}`]);
```